br/pkg/lightning/restore/get_pre_info.go

```diff
@@ -545,26 +556,35 @@ func (p *PreRestoreInfoGetterImpl) EstimateSourceDataSize(ctx context.Context, o
 					tbl.IndexRatio = sampledIndexRatio
 					tbl.IsRowOrdered = isRowOrderedFromSample
 
+					// if sample data failed due to max-error, fallback to use source size
```

Comment:
Can `tbl.IndexRatio` be zero?

Reply:
Seems it is a bug before....

Reply:
When `sampleDataFromTable` failed, `sampledIndexRatio/tbl.IndexRatio` will be zero.